website/source/docs/auth/aws-ec2.html.md

```diff
@@ -512,6 +512,17 @@ The response will be in JSON. For example:
         AWS Public key required to verify PKCS7 signature of the EC2 instance metadata.
       </li>
     </ul>
+    <ul>
+      <li>
+        <span class="param">type</span>
+        <span class="param-flags">required</span>
+Takes the value of either "pkcs7" or "identity", indicating the type of
+document which can be verified using the given certificate. The reason is that
+the PKCS#7 document will have a DSA digest and the identity signature will have
```

Comment:
I think this is just a little cleaner if you cut "The reason is that" and just start from this line.